### PR TITLE
[REC-2023] Updating sponsors, venue, program, speakers and details.

### PR DESCRIPTION
**Pull request by fabioqcorreia**
[REC-2023] Updating general information

- Updated sponsors;
- Updated venue information;
- Included program;
- Included speakers;
- Updated major details;

### CODE REVIEW

File: content/events/2023-recife/local.md [hunk 1]
```diff
@@ -8,7 +8,12 @@ Description = "Local - Devopsdays Recife 2023"
 
 <h1>Local do evento</h1>
 <br>
+<<<<<<< HEAD
```
Comment (adrianmoisey) [inline content/events/2023-recife/local.md:11]: There is some cruft left over from a merge that needs to be removed

Comment (adrianmoisey) [reply content/events/2023-recife/local.md:11]: These seem to be causing the builds to fail

**Comment by adrianmoisey** [comment]
Adding `do-not-merge` as the builds are currently failing

File: content/events/2023-recife/local.md (hunk 1 above)
Comment (fabioqcorreia) [reply content/events/2023-recife/local.md:11]: Thanks! Will fix it.

**Comment by fabioqcorreia** [comment]
I think everything is fine. After this being approved, I will create 2024 edition PR.

**Comment by fabioqcorreia** [comment]
Hi, can someone please merge this so we can send 2024 details?

**Pull request merged**